.prettierrc.yml

```diff
@@ -0,0 +1,5 @@
+printWidth: 100
+semi: true
+singleQuote: true
+trailingComma: all
+parser: typescript
```

Comment:
I insist on tabWidth: 4

Reply:
why not use tab symbols and let everyone set preferred value in their IDE?
(let the holy war begin!)
With prettier it is rather simple - it will insert all the missing spaces. I don't like tabs because you can click "inside" of the tab and then you need to waste time moving the cursor

Reply:
Let's use the current value of 4 spaces as a compromise.